teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# creating a Person subclass: teacher
+class Teacher < Person
+  def initialize(specialization = 'Unknown', name = 'Unknown', age = 0, parent_permission: true)
```

Comment:
- [ ] Kindly refactor the constructor to follow the needed parameters that a Teacher will receive. ``age, specialization, name = "Unknown", parent_permission = true`` The only ones that are defined as default are `name` and `parent_permission`